### DISCUSSION TITLE
Geometrie Editieren in WFS-Thema

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#22] [Q&A]
Wie kann in WebGIS5 Editieren für WFS-Themen definiert werden? Die Option wird den Eigenschaften nicht angeboten.

**Comment by gis-eni** [comment]
Themen können in WebGIS 5 über FeatureServices (AGS, gView) editiert werden. Für AGS MapServer Dienste stehen dafür im CMS unter **Editing** Methoden zum einrichten von Editthemen bereit.

Ein Bearbeiten von Themen über WFS-T gibt es in WebGIS 1-5 bisher nicht!
